corehq/apps/locations/tests/test_location_fixtures.py

```diff
@@ -1,28 +1,97 @@
-from django.test import SimpleTestCase
+from datetime import datetime, timedelta
+from django.test import TestCase
 from corehq.apps.locations.models import SQLLocation, LocationType
-from ..fixtures import _location_to_fixture, _location_footprint
+from casexml.apps.phone.models import SyncLog
+from corehq.apps.users.models import CouchUser, CommCareUser
+from ..fixtures import _location_to_fixture, _location_footprint, should_sync_locations, fixture_last_modified
+from corehq.apps.fixtures.models import UserFixtureStatus, UserFixtureType
 
 
-class LocationFixturesTest(SimpleTestCase):
-
-    def test_metadata(self):
-        state = LocationType(
-            domain="test-domain",
+class LocationFixturesTest(TestCase):
+    def setUp(self):
+        self.domain = "Erebor"
+        self.username = "Durins Bane"
+        self.location_type = LocationType(
+            domain=self.domain,
             name="state",
             code="state",
         )
+        password = "What have I got in my pocket"
+        self.user = CommCareUser.create(self.domain, self.username, password)
+        self.user.save()
 
+    def tearDown(self):
+        all_users = CouchUser.all()
+        for user in all_users:
+            user.delete()
+        SQLLocation.objects.all().delete()
+        LocationType.objects.all().delete()
+        UserFixtureStatus.objects.all().delete()
```

Comment:
You might want to use [`delete_all_locations()`](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/locations/tests/util.py#L17-27) for the locations teardown - a bunch of random other crap gets created sometimes.  Also, it looks like you're not making an actual domain here, so it won't help you, but check out [`Domain.delete()`](https://github.com/dimagi/commcare-hq/blob/feb96ff3cfddf646e67fed42c7e3ed6d7cbd6641/corehq/apps/domain/models.py#L980-1000) if you're not already familiar - it does a cascading delete of a lot of things.